packages/react/src/components/Modal/index.tsx

```diff
@@ -62,19 +61,21 @@ export const Modal = forwardRef(function Modal(
     }
   }, [])
 
+  useEffect(() => {
+    document.body.style.overflow = isOpen ? 'hidden' : 'scroll'
+
+    return (): void => {
+      document.body.style.overflow = 'scroll'
```

Comment:
```suggestion
    document.body.style.overflow = isOpen ? 'hidden' : 'visible'

    return (): void => {
      document.body.style.overflow = 'visible'
```
기본값이 visible이어서 기본값으로 설정되면 좋을 것 같아용!!

Reply:
오홍 기본값이 scroll인줄 알았는데 아녔군여 
감사합니다 👍

Reply:
[90e79ea](https://github.com/price-offer/offer-design-system/pull/109/commits/90e79eaeb25314ec7078b7cf03fb1adfe34a2ff0) 홀리 붸에엥엥 ~